### PR TITLE
Dev tools modifications2

### PR DESCRIPTION
**Pull request by lisamburns**
Which other branches should this be merged with (if any)?

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -152,20 +152,21 @@ To see this application in your organization catalog, you must register new enti
 5. Navigate back to the homepage, and the catalog changes and entries will be visible for further inspection.
 
 ### <a id='iterate'></a>Iterate on your application
-
-#### **Set up your integrated development environment (IDE)**
-With your basic application workload working, you are ready to iterate on your application
-and test your code changes on the cluster.
-Tanzu Developer Tools for VSCode and VMware Tanzu’s official IDE extension for VSCode
-will assist you in developing and receiving feedback on the Tanzu Application Platform.
+=======
```
Comment (amakhija) [inline getting-started.md:155]: Leftover from merge

File: getting-started.md [hunk 2]
```diff
@@ -152,20 +152,21 @@ To see this application in your organization catalog, you must register new enti
...
+Now that you have a skeleton workload working, you are ready to iterate on your application
+and test code changes on the cluster.
+Tanzu Developer Tools for VSCode, VMware Tanzu’s official IDE extension for VSCode,
+helps you develop & receive fast feedback on your workloads running on the Tanzu Application Platform.
 
 The VSCode extension enables live updates of your application while running on the cluster,
 and allows you to debug your application directly on the cluster.
+=======
```
Comment (amakhija) [inline getting-started.md:163]: Also leftover from merge

File: getting-started.md [hunk 3]
```diff
@@ -152,20 +152,21 @@ To see this application in your organization catalog, you must register new enti
...
+For information about installing the pre-requisites and the Tanzu Developer Tools extension, see
+[Install Tanzu Dev Tools for VSCode](vscode-extension/install.md).
 
-For information about installing the prerequisites and the Tanzu Developer Tools extension, see
-[How to Install the VSCode Tanzu Extension](vscode-extension/install.md) for instructions.
 
 >**Note:** For this sample application, you must use Tilt v0.23.2 or a later version.
+=======
```
Comment (amakhija) [inline getting-started.md:169]: Leftover from merge

File: getting-started.md [hunk 4]
```diff
@@ -174,36 +175,41 @@ For information about installing the prerequisites and the Tanzu Developer Tools
...
  	* In the **Local Path** field, provide the path to the directory containing the Tanzu Java Web App.
  	* In the **Source Image** field, provide the destination image repository to publish an image containing your workload source code.
-	For example, `harbor.vmware.com/myteam/tanzu-java-web-app-source`.
+	For example, `gcr.io/myteam/tanzu-java-web-app-source`.
 
 You are now ready to iterate on your application.
 
 
 #### Live update your application
+=======
```
Comment (amakhija) [inline getting-started.md:184]: Leftover from merge

File: getting-started.md (hunk 1 above)
Comment (lisamburns) [reply getting-started.md]: addressed

**Pull request merged**